service-mesh/go.mod

```diff
@@ -2,9 +2,12 @@ module github.com/edgelesssys/contrast/service-mesh
 
 go 1.23.0
 
+replace github.com/edgelesssys/contrast => ..
```

Comment:
needs update of renovate config!

Reply:
Does it? We already exclude dependencies on the contrast module, what am I missing?